lib/KeyGenUtils.py

```diff
@@ -1,11 +1,91 @@
-import random
-import string
-import socket
+import uuid
+import getpass
+import Checksum
 
-def randomize(stringLength = 10):
-    lettersAndDigits = string.ascii_letters + string.digits
-    return ''.join(random.choice(lettersAndDigits) for i in range(stringLength))
+# Every Time Key will be different but gets verify only from the source it generated.
+# Key have the length of 108 characters ends with '=' symbol.
+# Please install pycryptodome before run this file. Use cmd 'pip install pycryptodome'.
+# Change the folder to Crypto from crypto.
 
 
-#def generateHash(systemIP, MAC):
-    #generate Hash from random String and IP of the system
\ No newline at end of file
+def __get_mac__():
+    mac = hex(uuid.getnode())
+    return mac
+
+def __get_current_user__():
+    username = getpass.getuser()
+    return username
+
+
+#One System -> One Key
```

Comment:
Add comments before every library function specifying, what is the purpose of the function in one line, then what are the params passed into it with there data types and also mention what is the returned data with its data type.